build_for_ultimaker.sh

```diff
@@ -43,13 +43,138 @@ DEB_DIR=`pwd`/debian
 KCONFIG=`pwd`/configs/${BUILDCONFIG}_config
 KERNEL_BUILD=`pwd`/_build_armhf/${BUILDCONFIG}-linux
 
+INITRAMFS_MODULES="sunxi_wdt.ko"
+INITRAMFS_COMPRESSION="${INITRAMFS_COMPRESSION:-.lzo}"
```

Comment:
Why use parameter substitution to default value, this implies that somewhere else the variables could be set.

Reply:
It's optional, we could just initialize it to a default value, but this yields free extra flexibility if we want to set the parameters (as they are mirroring the kernel equivalents) from outside the script.
I have no other answer then 'why not' :)